### PR TITLE
Comments manager

### LINKED ISSUE #96: Add comments manager
Include search

### PR DESCRIPTION
**Pull request by bsuro10**
- Added Comments Index for Admin (with permissions) - _/Comments_
- Added 3 params search for comments
- Changed the template of a comment so make sure to check me in comments section in an article and Comments Index 

 - #96

### CODE REVIEW

**Comment by ghost** [comment]
I think there should be also in which article the comment is, in the manage comment page. 
And also do we need details view?

File: DGN/Controllers/CommentsController.cs [hunk 1]
```diff
@@ -136,5 +138,19 @@ private bool CommentExists(int id)
         {
             return _context.Comment.Any(e => e.Id == id);
         }
+
+        public async Task<IActionResult> Search(string body, string fullname, string username)
```
Comment (ghost) [inline DGN/Controllers/CommentsController.cs:143]: This should be with [Authorize]
in other functions we won't let anyone unauthorized to see comments, so we should not let here.

**Review by ghost — CHANGES_REQUESTED** [review]
Very good!
I'm not sure why it didn't let me do request changes above, so doing it now

**Comment by bsuro10** [comment]
> I think there should be also in which article the comment is, in the manage comment page.
> And also do we need details view?

Done.
We don't need details since there are not alot of details so I put everything on the table.

**Review by ghost — DISMISSED** [review]
Great!!

**Review by barBen — DISMISSED** [review]

**Review by barBen — APPROVED** [review]

**Review by origranot — APPROVED** [review]

**Pull request merged**